extras/cronwrapper

```diff
@@ -15,6 +15,27 @@
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
 
+sendSlackMessage() {
+    escapedText=$(cat ${1} | sed 's/"/\"/g' | sed "s/'/\'/g" )
```

Comment:
This would be a good place to look at using `printf` with the `%q` format specifier. (`help printf` if you're not familiar with it)

Reply:
I'm not familiar but I'll take a look, if it's simpler to escape the text then I'm all for it.